sorts/recursive_mergesort_array.py

```diff
@@ -0,0 +1,62 @@
+"""A merge sort which accepts an array as input and recursively splits an array in half and sorts and combines them. """
```

Comment:
Pre-commit failed because this line is too long. Here is a fix:

```suggestion
"""A merge sort which accepts an array as input and recursively
splits an array in half and sorts and combines them.
"""
```